lib/app/routes/static.rb

```diff
@@ -21,6 +21,18 @@ class Static < Core
         erb :"site/getting-started", locals: {active_languages: active_languages}
       end
 
+      get '/installing-cli' do
+        erb :"site/installing-cli", locals: {active_languages: active_languages}
+      end
+  
+      get '/create-assignment' do
+        erb :"site/create-assignment", locals: {active_languages: active_languages}
+      end
+
+      get '/submit-assignment' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.